action.yml

```diff
@@ -70,4 +69,4 @@ inputs:
 
 runs:
   using: node12
-  main: setup-gcloud/dist/index.js
+  main: dist-frozen/index.js
```

Comment:
we can remove `dist-frozen`. this should point to `dist/index.js`

Reply:
done